### PR TITLE
Integration with Bolide

### PR DESCRIPTION
**Pull request by Scriptoz**
- **Project Name**: Bolide

- **Project Website**: https://bolide.fi/

- **Description**: Integration with Bolide

- **Please check if the PR fulfills these requirements**

* [x] The projects name, twitter, URL etc have been filled out in `index.ts`
* [x] Tests for the new features have been added (for bug fixes / features)
* [x] Any required Logos are included in this PR

- **Logo**: https://invest.bolide.fi/img/BLID_token_logo_2000x2000.png

### CODE REVIEW

**Review by not-reed — CHANGES_REQUESTED** [review]
Looks great, thanks for the PR! If you could just change how the `formatUnits` is being imported in the `fetchUserPositions` calls, the rest looks good to me :+1:

File: apps/workshop/projects/Bolide/modules/StakingPools.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    const [[amount, rewardDebt]] = (await ethcallProvider.all([
+      contract.userInfo(pid, user),
+    ])) as BigNumber[][];
+
+    const blidToken = pool.supplied[0].token;
+
+    const supplied = [
+      {
+        ...pool.supplied[0],
+        balance: parseFloat(formatUnits(amount, blidToken.decimals)),
```
[the remark this answers is not in the document]
Comment (Scriptoz) [reply apps/workshop/projects/Bolide/modules/StakingPools.ts]: @not-reed Thank you for the advise! Fixed

[the remark this answers is not in the document]
Comment (not-reed) [reply apps/workshop/projects/Bolide/modules/StakingPools.ts]: @Scriptoz awesome, thanks!

**Comment by Scriptoz** [comment]
@not-reed Issues with linter was fixed

**Pull request merged**